README.md

```diff
@@ -1,6 +1,37 @@
-# python-bioimage-io
-Python specific core utilities for working with the model zoo.
+# python-bioimage-io core
 
+Python specific core utilities for working with the BioimageIO model zoo.
 
-# Model Specification
-The model specification and its validation tools have moved to https://github.com/bioimage-io/spec-bioimage-io.
+## Installation
+
+The `bioimageio.core` package supports various back-ends for running BioimageIO networks:
+
+* Pytorch/Torchscript:
+  ```bash
+  # cpu installation (if you don't have an nvidia graphics card)
+  conda install -c pytorch -c conda-forge -c ilastik-forge bioimageio.core pytorch torchvision cpuonly
```

Comment:
Why do we need `ilastik-forge` in the channels here? Same for the conda install commands below.

Reply:
Not really once we have the package on conda-forge. Until then we could have some versions on ilastik-forge. Not a very strong opinion there. But this makes it easier to remove certain versions in the beginning. (I of course know of such mechanisms in the conda-forge community, but would not want to put too much stress on the community. I'd prefer to come to a somewhat stable api first).